compiler/AST/baseAST.cpp

```diff
@@ -76,7 +76,7 @@ void printStatistics(const char* pass) {
   int nExpr = nUnresolvedSymExpr + nSymExpr + nDefExpr + nCallExpr +
     nContextCallExpr + nNamedExpr;
   int kExpr = kUnresolvedSymExpr + kSymExpr + kDefExpr + kCallExpr +
-    kContextCallExpr + kNamedExpr;
+    kContextCallExpr + kForallExpr + kNamedExpr;
```

Comment:
I don't understand the purpose of kExpr or kForallExpr but I wonder if nStmt and nExpr should match the values added in kStmt and kExpr (with the ks replaced by ns)